File: Javascript/8kyu/even-or-odd.js
// Descritpion:
// Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.

// My Solution:
function evenOrOdd(number) {
    return number % 2 === 0 ? 'Even' : 'Odd';
  }

// Other solutions:
function evenOrOdd(number) {
    if (number % 2 === 0) {
      return "Even";
    }
    else {
      return "Odd"
    }
  }

// Another one:
const evenOrOdd = (number) => number % 2 ? "Odd" : 'Even'; 

File: Python/8kyu/even-or-odd.py
#Describtion:
# Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.

#Solution:
def even_or_odd(number):
    return "Even" if number % 2 == 0 else "Odd"

#Other solutions:
def even_or_odd(number):
  if number % 2 == 0:
    return "Even"
  else:
    return "Odd"